plutus-ledger-api/src/PlutusLedgerApi/Common/ProtocolVersions.hs

```diff
@@ -86,16 +94,26 @@ knownPVs =
     , valentinePV
     , changPV
     , plominPV
+    , anonPV
     ]
 
--- | This is a placeholder for when we don't yet know what protocol version will
--- be used for something. It's a very high protocol version that should never
--- appear in reality.  New builtins should always be given this protocol version
--- until they've been finalised.
---
--- We should not assign names to future protocol versions until it's
--- confirmed that they are correct, otherwise we could accidentally
--- associate something with the wrong protocol version.
+-- We're sometimes in an intermediate state where we've added new builtins but
+-- not yet released them (but intend to). This is used by some of the tests to
+-- decide what PVs the test should include.  UPDATE THIS when we're expecting to
+-- release new builtins in a forthcoming PV.
+newestPV :: MajorProtocolVersion
+newestPV = anonPV
```

Comment:
If new builtins are assigned to `futurePV` first, then changed to the appropriate PV when ready, then I don't see why we need this.